src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -43,7 +43,7 @@ class CSVUtils {
         if (StringUtils.isBlank(s)) {
             throw new IllegalArgumentException("${fieldName} is empty or unset")
         }
-        DecimalFormat format = DecimalFormat.getNumberInstance(LocalizationUtil.localizationService.currentLocale)
+        DecimalFormat format = DecimalFormat.getNumberInstance(LocalizationUtil.localizationService.currentLocale) as DecimalFormat
```

Comment:
I missed this the first time around, but this factory method might get expensive over a large data set. We can leave it for now, but we should figure out a better way of handling the repeated instantiation of a formatter (i.e. singleton instances with caching by locale?).